docs/quickstart.md

```diff
@@ -1,225 +1,88 @@
-# Quickstart
+# Getting started 
 
-## Prerequisites
-
-You need to have Python 3.7+ and Git available in your machine to be able to use the `ucc-gen` command.
-
-> Git is used to generate the add-on version from Git tags. Alternatively, you can use the `--ta-version` parameter.
-To be able to create an add-on using the UCC framework, you need to have at least:
-
-* a `globalConfig` file (in `JSON` or `YAML` format, `JSON` is mostly used).
-* a `package` folder.
-* `app.manifest` in the `package` folder ([documentation here](https://dev.splunk.com/enterprise/reference/packagingtoolkit/pkgtoolkitappmanifest/)).
-
-The `app.manifest` file now is being validated. See [Splunk Packaging Toolkit app.manifest schema definition](https://dev.splunk.com/enterprise/reference/packagingtoolkit/pkgtoolkitappmanifest/#JSON-schema-200) for more details.
-
-> If both the globalConfig.json and globalConfig.yaml files are present, then the globalConfig.json file will take precedence.
-
-The JSON schema for the `globalConfig` file can be found in the `splunk_add_on_ucc_framework/schema/schema.json` file.
-
-### Add-on naming convention
-
-See [Naming conventions for apps and add-ons in Splunkbase](https://dev.splunk.com/enterprise/docs/releaseapps/splunkbase/namingguidelines/)
-for help naming your add-on.
+Install the UCC framework and start building your first add-on. Then you can build new add-ons from the existing ones.
 
-## Initialize new add-on
-
-> Initialization of the new add-on is available from version `5.19.0` and up of `ucc-gen`.
-
-The following commands are macOS and Linux specific.
-
-* Set up and activate the Python virtual environment:
-
-```bash
-python3 -m venv .venv
-source .venv/bin/activate
-```
-
-* Install `splunk-add-on-ucc-framework`:
-
-```bash
-pip install splunk-add-on-ucc-framework
-```
-
-* Initialize the new add-on:
-
-```bash
-ucc-gen init --addon-name "demo_addon_for_splunk" --addon-display-name "Demo Add-on for Splunk" --addon-input-name demo_input
-```
+## Prerequisites
 
-The new add-on is located in the `demo_addon_for_splunk` folder and can be built using
-the [the following commands](#build-the-already-existing-add-on):
+Make sure that the following software is installed on your machine:
```

Comment:
```suggestion
Make sure that the following software is installed on your machine:

```

Reply:
(empty line)

otherwise it renders like this - 
<img width="869" alt="image" src="https://github.com/user-attachments/assets/5b24ec47-d733-4b49-983b-360eeae177bc">